src/main/resources/com/cognifide/gradle/aem/environment/README.md

```diff
@@ -0,0 +1,94 @@
+
+# Local AEM Environment Setup with Docker
```

Comment:
I prefer still one README.md for all GAP features. 

if we decide to split it, then we probably need to do it for all features;

"no matter how as long as consistent"  once again :)